bw2io/strategies/generic.py

```diff
@@ -11,6 +11,36 @@
 
 
 def format_nonunique_key_error(obj, fields, others):
+    """
+    This function takes an object that can't be uniquely linked to the target
```

Comment:
...same issue as [here](https://github.com/brightway-lca/brightway2-io/pull/175#discussion_r1153681421): You mix imperative ("Perform XYZ") and declarative ("This function performs XYZ") throughout.

Reply:
this is resolved on the last changes, I made the PR and forgot to push some local changes after